frontend/libs/studio-components/src/components/StudioTextarea/StudioTextarea.test.tsx

```diff
@@ -0,0 +1,143 @@
+import type { StudioTextareaProps } from './StudioTextarea';
+import { StudioTextarea } from './StudioTextarea';
+import { act, render, screen } from '@testing-library/react';
+import type { RefObject } from 'react';
+import React from 'react';
+import userEvent from '@testing-library/user-event';
+
+describe('StudioTextarea', () => {
+  it('Renders a textarea', () => {
+    renderTextarea();
+    expect(screen.getByRole('textbox')).toBeInTheDocument();
+  });
+
+  it('Renders with the given value', () => {
+    const value = 'test';
+    renderTextarea({ value });
+    expect(screen.getByRole('textbox')).toHaveValue(value);
+  });
+
+  it('Updates the value when the component rerenders with another value', () => {
+    const value = 'test';
+    const { rerender } = renderTextarea({ value });
+    expect(screen.getByRole('textbox')).toHaveValue(value);
+    const newValue = 'new value';
+    rerender(<StudioTextarea value={newValue} />);
+    expect(screen.getByRole('textbox')).toHaveValue(newValue);
+  });
+
+  it('Updates the value when the user types', async () => {
+    const user = userEvent.setup();
+    renderTextarea();
+    const textarea = screen.getByRole('textbox');
+    const newValue = 'new value';
+    await act(() => user.type(textarea, newValue));
+    expect(textarea).toHaveValue(newValue);
+  });
+
+  it('Calls the onChange handler when the user types', async () => {
+    const user = userEvent.setup();
+    const onChange = jest.fn();
+    renderTextarea({ onChange });
+    const textarea = screen.getByRole('textbox');
+    const newValue = 'new value';
+    await act(() => user.type(textarea, newValue));
+    expect(onChange).toHaveBeenCalledTimes(newValue.length);
+    const expectedTarget = expect.objectContaining({ value: newValue });
+    const expectedEvent = expect.objectContaining({ target: expectedTarget });
+    expect(onChange).toHaveBeenLastCalledWith(expectedEvent);
+  });
+
+  it('Calls the onBlur handler when the user blurs', async () => {
+    const user = userEvent.setup();
+    const onBlur = jest.fn();
+    renderTextarea({ onBlur });
+    const textarea = screen.getByRole('textbox');
+    await act(() => user.click(textarea));
+    await act(() => user.tab());
+    expect(onBlur).toHaveBeenCalledTimes(1);
+  });
+
+  it('Does not display the after blur error message by default', () => {
```

Comment:
This comment goes for all the testcases (also for `textField`); could we wrap the _after blur_ in double quotes so the test cases are easier to read? 😅 Or something similar

Reply:
I'll se what I can do.